aderyn_core/src/detect/detector.rs

```diff
@@ -119,6 +119,7 @@ pub fn get_detector_by_name(detector_name: &str) -> Box<dyn Detector> {
 
 #[derive(Debug, PartialEq)]
 pub enum IssueSeverity {
+    Utility,
```

Comment:
Should we call this a different name? (cc: @ChaseTheLight01, @TilakMaddy )

Reply:
I haven't gotten a chance to review this wholly but I was wondering if it's possible to have a separate trait itself like DetectorUtility or something with its own severity enumerations (or maybe verbosity enumerations) ....

Reply:
in the future, we would have separate detector-esque traits anyways for vyper ast . So.

Reply:
It may make it harder for us to manage issue severities across all detector traits

Reply:
Yes, that makes sense!

Ooooo, I like the idea of a separate impl. Within `nyth` we can then have different commands for creating report detectors vs reusable detectors

Reply:
Creating a generate subcommand for nyth to add resuable detectors 

Todo : Add to Milestone